cartscript.js

```diff
@@ -0,0 +1,77 @@
+
+const cart=document.querySelector("cart")
+const cartRemove_btns= document.querySelectorAll(".trash-btn");
+addCartRemoveEvent();
+changingItemQuantityEvent();
+itemLikeEvent();
+
+///////// deleting item from a cart
+function addCartRemoveEvent(){
+    cartRemove_btns.forEach( function(btn) {
+        btn.addEventListener("click", handleRemoveCart);
+    })
+}
+
+///// changing quantity of an item in the cart
+function changingItemQuantityEvent(){
+    let cartQuantities= document.querySelectorAll(".cart-quantity");
+    cartQuantities.forEach(function(cartQuantity){
+        cartQuantity.addEventListener("change" , handle_ChangeItemQuantity);
+    })
+}
+
+//// clicking like button or heart Icon
+function itemLikeEvent(){
+    let heartBtns = document.querySelectorAll(".like-btn");
+    heartBtns.forEach( function(heartBtn){
+       // console.log(heartBtn.value);
+        heartBtn.addEventListener("click", handle_LikeItem);
+    })
+}
+function handle_LikeItem(){
+    let c=this.className;
+    console.log(c);
+    if( c == "fa fa-heart like-btn notLiked"){
+         this.style.color= "red";
+         this.className="fa fa-heart like-btn Liked";
+    }
+    else{
+        this.style.color= "white";
+        this.className="fa fa-heart like-btn notLiked";
+    }
+    
+  
+    
+}
+function handle_ChangeItemQuantity(){
+  if(this.value < 1 ){
+    this.value=1;
+  }
+  updateCartTotal();
+}
+
+ function handleRemoveCart(){
+    console.log("handling cart deletion");
+    let parent=this.parentElement.parentElement;
+    console.log(parent);
+    parent.remove();
+    updateCartTotal();
+ }
+
+function updateCartTotal(){
+    let cartItems= document.querySelectorAll(".cart-item");
```

Comment:
Again, the `cartItems` variable does not get reassigned so it's better to define it using `const`.